main.go

```diff
@@ -42,6 +48,14 @@ type kubeRulesLoader struct {
 	managedTenants string
 }
 
+func (k *kubeRulesLoader) getLokiAlertingRules() ([]*lokiv1beta1.AlertingRule, error) {
+	return nil, nil
```

Comment:
I guess this is TODO?

Reply:
Yes I need to figure out a simple way to create a kubeclient w/o import the whole k/k universe here.